docs_outdoornav_user_manual/web_user_interface/ui_autonomous_mode.mdx

```diff
@@ -161,34 +161,34 @@ To add a Task to the end of a Mission:
     replaced accordingly depending on the task selected (waypoint icons will keep the colours
     assigned to them based on placement).
 
-    -   **Dock UGV:** <img src="/img/outdoornav_images/dockTask_40px.png" width="30px"/>
+    -   <img src="/img/outdoornav_images/dockTask_40px.png" width="30px"/> <strong>Dock UGV: </strong>
         Will dock the UGV to begin charging the UGVs
         battery. See [Autonomous Docking](#autonomous-docking)
         for more information on the autonomous docking feature.
 
-    -   **Move PTZ:** <img src="/img/outdoornav_images/cameraTask_40px.png" width="30px"/>
+    -   <img src="/img/outdoornav_images/cameraTask_40px.png" width="30px"/>  <strong>Move PTZ: </strong>
         Will move the PTZ camera to the position selected
         in the task settings.
 
         Settings: Select the camera position. See
         [Pan-Tilt-Zoom (PTZ) View](#ptz-view) for details on how to
         save camera positions.
 
-    -   **Save Image:** <img src="/img/outdoornav_images/cameraTask_40px.png" width="30px"/>
+    -   <img src="/img/outdoornav_images/cameraTask_40px.png" width="30px"/>  <strong>Save Image: </strong>
         Will save an image using one of the UGV camera(s)
         to the **/onav_log** folder and can be retrieved using a tool
         such as Filezilla.
 
         Settings: Select which camera the image will be saved from.
 
-    -   **Start/Stop Video Recording:** <img src="/img/outdoornav_images/recordTask_40px.png" width="30px"/>
+    -   <img src="/img/outdoornav_images/recordTask_40px.png" width="30px"/> <strong>Start/Stop Video Recording: </strong>
         Will start/stop recording video using one of the
         UGV camera(s) to the **/onav_log** folder and can be retrieved
         using a tool such as Filezilla.
 
         Settings: Select which camera the recording will come from.
 
-    -   **Undock UGV:** <img src="/img/outdoornav_images/undockTask_40px.png" width="30px"/>
+    -   <img src="/img/outdoornav_images/undockTask_40px.png" width="30px"/> <strong>Undock UGV: </strong>
         Will undock the UGV from the autocharge dock. Once
         completed, the UGV can be sent on autonomous missions. It is
         often recommended to place the undock Task first in the list of Waypoints;

```

Comment:
Should undock be capitalized, or some other font formatting to draw attention? It seems odd that Task is capitalized.

Reply:
Fixed the task capitalization and I think that address the issue